itsim/datastore/datastore.py

```diff
@@ -1,16 +1,16 @@
 from abc import abstractmethod
```

Comment:
Add this import if you go with my `mkstemp` suggestion below.
```suggestion
from abc import abstractmethod
import tempfile
```